kubernetes-client/src/test/java/io/fabric8/kubernetes/client/ConfigTest.java

```diff
@@ -195,6 +209,36 @@ public void testWithKubeConfigAndSytemPropertiesAndBuilder() {
     assertEquals("testns2", config.getNamespace());
   }
 
+  @Test
+  public void testWithNamespacePath() {
```

Comment:
In reality all cases are "with namespace path" but the ones that explicitly specify a namespace via system/env.

This is an other reason why I prefer the default namespace path to be overridden via system/env as its easier to isolate.